addons/survey/migrations/11.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,26 @@
+---Fields in module 'survey'---
+survey       / survey.question          / type (selection)              : selection_keys is now '['date', 'free_text', 'matrix', 'multiple_choice', 'numerical_box', 'simple_choice', 'textbox']' ('['datetime', 'free_text', 'matrix', 'multiple_choice', 'numerical_box', 'simple_choice', 'textbox']')
+# Nothing to do: selections keys are the same
+
+survey       / survey.question          / validation_max_date (datetime): type is now 'date' ('datetime')
+DONE: pre-migration: Renamed to preserve data
```

Comment:
Include a comment saying that the field is then converted automatically to date.